libs/dmlf/tests/colearn_muddle/colearn_muddle.cpp

```diff
@@ -76,26 +76,16 @@ class LearnerTypedUpdates
   LearnerTypedUpdates(const std::string &pub, const std::string &priv, unsigned short int port,
```

Comment:
Const wrong way around

Also you could pass in the pub (and maybe the priv) as a copy and move it into pub_